input_name.py

```diff
@@ -9,30 +9,31 @@ class InputName(pygame.sprite.Sprite):
     def __init__(self):
         super().__init__()
         #Instruction and re-enter name strings
-        self.instruction_string = "Please input your name(must be between 1 and 15 characters) below:"
+        self.instruction_string = "Please input your name(must be between 1-15 characters and have no spaces) below:"
```

Comment:
Maybe we can say max 15 characters and no spaces to shorten the description.

Reply:
True we can fix that if we want, just didn't want someone to be a smart ass and try 0 since that is less than 15 and complain that it wouldn't let them haha